cmd/viam-agent/main_windows.go

```diff
@@ -51,18 +56,24 @@ func main() {
 	if err != nil {
 		return
 	}
-	defer elog.Close()
+	defer func() {
+		//nolint:errcheck,gosec // logging utility errors can be ignored
+		elog.Close()
+	}()
```

Comment:
```suggestion
	defer func() {
		goutils.UncheckedError(elog.Close())
	}()
```
Optional! 

Can use UncheckedError for errors we (generally) don't care about, as it'll still force a log line out in the rare cases an error happens. Same goes for similar lines elsewhere.